pkg/upgrader/upgrader_v1alpha1.go

```diff
@@ -148,7 +148,7 @@ func (v1alpha1 V1Alpha1) Upgrade(node corev1.Node, tag string) (err error) {
 	// TODO(andrewrynhard): Request upgrade with context timeout.
 	ctx := context.TODO()
 
-	version, err := getVersion(ctx, c)
+	version, err := getVersion(ctx, c, 2*time.Minute)
 	if err != nil {
```

Comment:
since we already have context, does it make sense to use context with a deadline?

Reply:
I'm gonna actually close this. This isn't the right approach come to find out.